src/it/MANTRUN-221/pom.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.apache.maven.plugins.antrun</groupId>
+  <artifactId>MANTRUN-221</artifactId>
+  <packaging>pom</packaging>
+  <version>1.0</version>
+  <url>https://issues.apache.org/jira/browse/MANTRUN-221</url>
+  <description>
+    Checks that properties defined at build time are passed to Ant invocation
+  </description>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>ca.szc.maven</groupId>
+        <artifactId>jsonpath-maven-plugin</artifactId>
```

Comment:
Is it really needed to use this third party plugin for this test?

My concern is that when you run tests you download stuff from the Internet.
If it is not needed I would like to not use it

Reply:
I'm not aware of any way to set up the specific conditions that trigger this bug without a plugin of some kind. Defining a new property on the command line adds it to both user and project.

If no external dependencies is a requirement, the only way would be to create a minimal plugin inside the maven-antrun-plugin's source tree that does nothing but add properties to `session.getUserProperties()`. This plugin would need to be built when the integration test profile is active, and before the test for this bug is run. Wouldn't be very clean, but is possible.

Reply:
Maybe it is not worth to create a new minimal plugin just for a test